RFC/src/RFC-0201_TariScript.md

```diff
@@ -484,145 +544,178 @@ Alice owns \\( C_a \\) and provides the required script to spend the UTXO as was
+| Transaction input | Symbols                               | Knowledge                                                                                     |
+|-------------------|---------------------------------------|-----------------------------------------------------------------------------------------------|
+| commitment        | \\( C_a = k_a \cdot G + v \cdot H \\) | Alice knows the blinding factor and value                                                     |
+| features          | \\( F_a \\)                           | Public                                                                                        |
+| script            | \\( \alpha_a \\)                      | Public                                                                                        |
+| script input      | \\( \input_a \\)                      | Public                                                                                        |
+| height            | \\( h_a \\)                           | Public                                                                                        |
+| script signature  | \\( s_{S1a},s_{S2a}, R_{Sa} \\)       | Alice knows \\( k_{Sa},\\, r_{Sa} \\) and \\( k_{a},\\, v_{a} \\) of the commitment \\(C_a\\) |
+| offset public key | \\( K_{Oa} \\)                        | Not used in this transaction                                                                  |
 
 | Transaction output | Symbols                               | Knowledge                                                              |
-|:-------------------|:--------------------------------------|:-----------------------------------------------------------------------|
-| commitment         | \\( C_b = k_b \cdot G + v \cdot H \\) | Alice and Bob know the spend key and value                             |
+|--------------------|---------------------------------------|------------------------------------------------------------------------|
+| commitment         | \\( C_b = k_b \cdot G + v \cdot H \\) | Alice and Bob know the blinding factor and value                       |
 | features           | \\( F_b \\)                           | Public                                                                 |
-| script hash        | \\( \scripthash_b \\)                 | Script is effectively public. Only Bob knows the correct script input. |
+| script             | \\( \script_b \\)                     | Script is public. Only Bob knows the correct script input.             |
 | range proof        |                                       | Alice and Bob know opening parameters                                  |
 | offset public key  | \\( K_{Ob} \\)                        | Alice knows \\( k_{Ob} \\)                                             |
+| sender signature   | \\( s_{Mb}, R_{Mb} \\)                | Alice knows \\( k_{Ob} \\)  and the metadata)                          |
```

Comment:
```suggestion
| sender metadata signature   | \\( s_{Mb}, R_{Mb} \\)                | Alice knows \\( k_{Ob} \\)  and the metadata                          |
```

Reply:
I would add the contents of the challenge here

Reply:
The challenge contents are all public information, these tables are for private information.